packages/service-library/setup.py

```diff
@@ -17,15 +18,14 @@ def read_reqs(reqs_path: Path):
     here / "requirements" / "_base.in"
 )  # WEAK requirements
 
-aiohttp_requirements = read_reqs(
-    here / "requirements" / "_aiohttp.in"
-)  # WEAK requirements
-
 
 test_requirements = read_reqs(
     here / "requirements" / "_test.txt"
 )  # STRONG requirements
 
+aiohttp_requirements = read_reqs(here / "requirements" / "_aiohttp.in")
```

Comment:
could you please add a comment explaining why the `*.in` files are included here?

Reply:
Most of the info is in [requirements/python-dependencies.md)](https://github.com/ITISFoundation/osparc-simcore/blob/d9125d4e44d63a5413bac78ff7c547c3e6d37dd7/requirements/python-dependencies.md) but i will extend it.

Basically, the idea is that *libraries* shall have *weak* constraint requirements while *services* shall have *hard* constraints (i.e. frozen).  Weak requirements are in the ``*.in`` files while *hard* are in the ``*.txt`` files.

The main reason is that the former are shared among different services and provides more degrees of freedom. In the case of the services, those are final and there is no need for those degrees of freedom.

Libraries requirements are only frozen for testing (therefore ``test_requirements= .. txt`` in libraries setup).

Reply:
I had a hunch it was that. Thanks for clearing it up